common/search/src/main/scala/weco/api/search/config/builders/PipelineElasticClientBuilder.scala

```diff
@@ -6,15 +6,59 @@ import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueReques
 import weco.catalogue.display_model.PipelineClusterElasticConfig
 import weco.elasticsearch.ElasticClientBuilder
 
+/**
+  * Build an ElasticClient that will connect to the appropriate database
+  * as specified for this environment.
+  *
+  * The settings may come either from environment variables
+  * (if API_SETTINGS_MODE is environment) or from the AWS Secrets Manager
+  */
 object PipelineElasticClientBuilder {
+  def apply(serviceName: String): ElasticClient =
+    sys.env.get("API_SETTINGS_MODE") match {
+      case None | Some("secretsmanager") =>
+        SecretsElasticClientBuilder(serviceName)
```

Comment:
This is fine for now, but at some point I think we should update the API Terraform to pass this setting, so we can remove the default and always be explicit.

Reply:
Is it necessary to specify where the secrets come from? A lot of tools just use a predefined precedence (ie here we'd get from environment or, if that wasn't present, secrets manager)